libs/androidlib/src/mill/androidlib/AndroidAppKotlinModule.scala

```diff
@@ -26,25 +26,34 @@ import upickle.implicits.namedTuples.default.given
 @mill.api.experimental
 trait AndroidAppKotlinModule extends AndroidKotlinModule with AndroidAppModule { outer =>
 
-  def kotlinSources = Task.Sources("src/main/kotlin")
+  def kotlinSources: T[Seq[PathRef]] = Task.Sources("src/main/kotlin")
   override def sources: T[Seq[PathRef]] =
     super[AndroidAppModule].sources() ++ kotlinSources()
 
-  trait AndroidAppKotlinTests extends KotlinTests with AndroidAppTests {
-    def kotlinSources = Task.Sources("src/test/kotlin")
+  trait AndroidAppKotlinTests extends AndroidAppKotlinModule with AndroidAppTests {
+    override def kotlinVersion: T[String] = outer.kotlinVersion
+
+    override def kotlinSources: T[Seq[PathRef]] = Task.Sources("src/test/kotlin")
+
     override def sources: T[Seq[PathRef]] =
       super[AndroidAppTests].sources() ++ kotlinSources()
+
+    override def kotlincPluginMvnDeps: T[Seq[Dep]] = outer.kotlincPluginMvnDeps()
   }
 
   trait AndroidAppKotlinInstrumentedTests extends AndroidAppKotlinModule
       with AndroidAppInstrumentedTests {
 
-    override final def kotlinVersion = outer.kotlinVersion
-    override final def androidSdkModule = outer.androidSdkModule
+    override final def kotlinVersion: T[String] = outer.kotlinVersion
+    override final def androidSdkModule: ModuleRef[AndroidSdkModule] = outer.androidSdkModule
+
+    override def kotlinSources: T[Seq[PathRef]] = Task.Sources("src/androidTest/kotlin")
```

Comment:
I think `kotlinSources` is meant to be private and should therefore not be overridden.

Reply:
Thanks, I made it private instead then